features/schools/onboarding/descriptions.feature

```diff
@@ -20,14 +20,14 @@ Feature: Description
 
   Scenario: Page title
     Given I am on the 'description' page
-    Then the page title should be 'Enter a short description of your school'
+    Then the page title should be 'Write a short description of your school'
```

Comment:
As mentioned previously, I think we should leave this for when we have dev help and redo the whole question.